source/tomopy/misc/corr.py

```diff
@@ -598,6 +598,9 @@ def remove_ring(rec, center_x=None, center_y=None, thresh=300.0,
     else:
         raise ValueError("int_mode should be WRAP or REFLECT")
 
+    if not 0 <= theta_min <= 360:
+        raise ValueError("theta_min should be in the range 0 - 360")
+
```

Comment:
My understanding is that `theta_min` in the python API is the same a `angular_min` in the C API. Therefore, `theta_min` determines `m_azi`. I agree that it is sensible to check that `theta_min` must be <= zero and < 360 because a ring artifact cannot be longer than 360 degrees. However, is `theta_min` = 360 valid?